Documentation/Scripts/generateMdFiles.py

```diff
@@ -110,9 +110,7 @@ def getRestBodyParam():
         if thisVerb['parameters'][nParam]['in'] == 'body':
             descOffset = thisVerb['parameters'][nParam]['x-description-offset']
             addText = ''
-            if 'additionalProperties' in thisVerb['parameters'][nParam]['schema']:
-                addText = "free style json body"
```

Comment:
This phrase doesn't convey any useful information that isn't already provided by the prose description. Considering most bodies seem to get this additional text right now it's just a distraction and might in some cases actually be wrong. The underlying assumption is "if we can't represent the body in Swagger, it's free-form" -- which is obviously not reliable.